src/pages/index.tsx

```diff
@@ -51,7 +53,30 @@ function Page(props: Props) {
         Sections: Components imported from '../components/sections' only.
         Do not import or render components from any other folder in here.
       */}
-      <h1>{title}</h1>
+      <Hero>
+        <HeroContent>
+          <div>
+            <h1>New Products Available</h1>
```

Comment:
Since `HeroContent` behaves as a region role, it [should include a label that describes the purpose of the content](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/region_role#:~:text=should%20include%20a%20label%20that%20describes%20the%20purpose%20of%20the%20content%20in%20the%20region). We could link them up:

```suggestion
        <HeroContent aria-labelledby="hero-heading">
          <div>
            <h1 id="hero-heading">New Products Available</h1>
```